client/lib/signup/cart.js

```diff
@@ -11,7 +11,33 @@ var wpcom = require( 'lib/wp' ),
 	cartValues = require( 'lib/cart-values' ),
 	cartItems = cartValues.cartItems;
 
+function addProductsToCart( cart, newCartItems ) {
+	forEach( newCartItems, function( cartItem ) {
+		cartItem.extra = Object.assign( cartItem.extra || {}, {
+			context: 'signup'
+		} );
+		const addFunction = cartItems.add( cartItem );
+
+		cart = cartValues.fillInAllCartItemAttributes( addFunction( cart ), productsList.get() );
+	} );
+
+	return cart;
+}
+
 module.exports = {
+	createCart: function( cartKey, newCartItems, callback ) {
```

Comment:
What do you think about removing `addToCart` entirely and replacing it with this function? I don't see a reason for us ever to update an existing cart during `/start`.